kubernetes/data/state-channel-seedlist.csv

```diff
@@ -0,0 +1,10 @@
+DAG7QrKeoHbYezJ6YHUCt7PJDixWgxM4vfKz8qVT
+DAG3FJFFsjzYeSNVeL3cykLqWbijkq25K6Eg23yp
+DAG5kq5J6gudAbyGsgqPReGz8ZDRM94UoPwCFSem
+DAG3ysjCYZCWoTw3SKoK13McfdA5LZWYBr7ue47d
+DAG7cCCNgP8D5rdEznhSNXpAfQ2KhuScxhg1YhVX
+DAG2T7jB5kXpQMEpDErB2NcoQfeJP7DwbueK3A5i
+DAG16Q8MxAvihBbFk8wuSaCVzieq75zZA3mh414r
+DAG6wD9h8NE6qDuVT8ZtJFaU3ZzD9BqBjWD8GDee
+DAG0R1CMwfLPNp4qkA2TTqkcKWp3mC7U4ZMD5n1B
+DAG7gcb396YoLVzNhCEF2nrB79uwV3wr4BvXSU9f
```

Comment:
are you sure it should be a DAGAddress, not a node id (so the public key)?

Reply:
Yes it needs to be a DAGAddress. Currently it's derived from public key, but we will add functionality to derive the address from state channel genesis.